layouts/prepare-registration.vue

```diff
@@ -1,54 +1,190 @@
 <script setup lang="ts">
-import { useForm } from "vee-validate";
-import * as yup from "yup";
+interface Question {
+    id: number;
+    question: string;
+}
 
-const schema = yup.object({
-    registrationUrl: yup.string().required(),
-});
+const questions: Ref<Question[]> = ref([]);
+const editingQuestion = ref<Question | null>(null);
+const confirm = useConfirm();
+const toast = useToast();
 
-const { defineField, handleSubmit, resetForm, errors } = useForm({
-    validationSchema: schema,
-});
+const onSubmit = (e: Event) => {
+    e.preventDefault();
+    console.log("values: ", questions.value);
```

Comment:
Is important to log to console? @JeromeTana

Reply:
will fix in next PR